pkg/oak_run.go

```diff
@@ -16,29 +16,35 @@ func NewOakRun(vendorManager *VendorManager) *OakRun {
 
 	return &OakRun{
 		vendorManager: vendorManager,
-
-		DownloadURL: cv.GetString("vendor.oak_run.download_url"),
-		StorePath:   cv.GetString("vendor.oak_run.store_path"),
+		DownloadURL:   cv.GetString("vendor.oak_run.download_url"),
+		JarFile:       cv.GetString("vendor.oak_run.jar_file"),
+		StorePath:     cv.GetString("vendor.oak_run.store_path"),
 	}
 }
 
 type OakRun struct {
 	vendorManager *VendorManager
-
-	DownloadURL string
-	StorePath   string
+	DownloadURL   string
+	JarFile       string
+	StorePath     string
 }
 
 type OakRunLock struct {
 	DownloadURL string `yaml:"download_url"`
+	JarFile     string `yaml:"jar_file"`
 }
 
 func (or OakRun) Dir() string {
 	return or.vendorManager.aem.baseOpts.ToolDir + "/oak-run"
 }
 
 func (or OakRun) lock() osx.Lock[OakRunLock] {
-	return osx.NewLock(or.Dir()+"/lock/create.yml", func() (OakRunLock, error) { return OakRunLock{DownloadURL: or.DownloadURL}, nil })
+	return osx.NewLock(or.Dir()+"/lock/create.yml", func() (OakRunLock, error) {
+		return OakRunLock{
+			DownloadURL: or.DownloadURL,
+			JarFile:     or.JarFile,
```

Comment:
in java_manager.go the `homeDir` is not put into lock
I believe for oak run it should work the same.

now adding it will enforce to resetup oakrun after upgrading AEMC (but it shouldn't be required)